datasource-local/src/test/java/cmm/apps/esmorga/datasource_local/user/UserLocalDatasourceImplTest.kt

```diff
@@ -32,6 +35,25 @@ class UserLocalDatasourceImplTest {
         return dao
     }
 
+    private fun provideFakeSharedPreferences() : SharedPreferences {
+        val sharedPreferences = mockk<SharedPreferences>(relaxed = true)
+        coEvery { sharedPreferences.getString("refresh_token", null) } coAnswers {
+            fakeSharedToken
+        }
+        coEvery { sharedPreferences.getString("access_token", null) } coAnswers {
+            fakeSharedRefreshToken
+        }
+        coEvery { sharedPreferences.edit().putString("access_token", any()).apply() } coAnswers {
+            fakeSharedToken = "fakeToken"
```

Comment:
here you can use the `capture()` thing to store the value you pass and not "fakeToken" always